spec/vast_parser.spec.js

```diff
@@ -80,23 +80,32 @@ describe('VASTParser', () => {
         });
       });
 
-      it('applies url filters and saves url in parentURLs', () => {
+      it('applies url filters', (done) => {
         VastParser.URLTemplateFilters = [(url) => url.replace('foo', 'bar')];
 
-        return VastParser.fetchVAST('www.foo.foo').finally(() => {
-          expect(VastParser.parentURLs).toEqual(['www.bar.foo']);
+        const urlHandlerSpy = jest.spyOn(VastParser.urlHandler, 'get');
+
+        VastParser.fetchVAST('www.foo.foo').then(() => {
+          done();
+          expect(urlHandlerSpy).toHaveBeenCalledWith(
```

Comment:
You might want to put the done after the expect as Jest do in their documentation https://jestjs.io/docs/asynchronous#callbacks